packages/manager/modules/office/README.md

```diff
@@ -0,0 +1,29 @@
+# @ovh-ux/office
```

Comment:
To stay aligned with actual modules I think we should name it like:

- **@ovh-ux/manager-office**